src/storage/filesystem-adapter.test.ts

```
@@ -0,0 +1,150 @@
+    // Clean up test files after each test
+    try {
+      await fs.rm(testDir, { recursive: true, force: true });
+    } catch (error) {
+      // Ignore errors if directory doesn't exist
+    }
+  });
+
+  describe('constructor', () => {
+    it('should use default file path if not provided', () => {
+      const defaultAdapter = new FilesystemAdapter();
+      expect(defaultAdapter['filePath']).toBe(path.join(process.cwd(), 'run-cache-data.json'));
```

ESLint

[error] ["filePath"] is better written in dot notation.

(dot-notation)